### PR TITLE
Clarify a few things in Cassandra example.

### PR DESCRIPTION
**Pull request by goltermann**
Fixing Cassandra example.  #8356

Up for any suggestions on how to clean up the paragraph that starts "In Kubernetes, a Service..."  That's still not as clear as I'd like.

### CODE REVIEW

File: examples/cassandra/README.md [hunk 1]
```diff
@@ -204,11 +207,11 @@ cassandra-gnhk8     10.244.0.5                                              kube
 
 Notice that one of the pods has the human readable name ```cassandra``` that you specified in your config before, and one has a random string, since it was named by the replication controller.
 
-To prove that this all works, you can use the ```nodetool``` command to examine the status of the cluster, for example:
+To prove that this all works, you can use the ```nodetool``` command to examine the status of the cluster.  To do this, SSH to one of the worker VMs in your cluster.  Running ```sudo docker ps``` will help you find the \<cassandra-container-id\> used below.
```
Comment (mikedanese) [inline examples/cassandra/README.md]: Might want to adapt this to use kubectl exec at this point. From the master:
```
kubectl exec -ti cassandra -- nodetool status
```

**Comment by goltermann** [comment]
@mikedanese Fixed to use kubectl exec, and also updated the docs to use the v3 image (matching the yaml.

**Comment by mikedanese** [comment]
@goltermann nice. are you sure the yaml was correct? it doesn't look like the v3 tag is pushed.

https://registry.hub.docker.com/u/kubernetes/cassandra/tags/manage/

**Comment by goltermann** [comment]
@mikedanese They're gcr.io links in the yaml

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/cassandra/cassandra.yaml

File: examples/cassandra/README.md [hunk 2]
```diff
@@ -27,7 +29,7 @@ spec:
     resources:
       limits:
         cpu: "1"
-    image: kubernetes/cassandra:v2
+    image: kubernetes/cassandra:v3
```
Comment (mikedanese) [inline examples/cassandra/README.md]: then these image urls should be gcr.io/google_containers/cassandra:v3 as well.

**Comment by goltermann** [comment]
@mikedanese Ah, gotcha.  Fixed in the yaml snippets and some of the kubectl outputs.

**Comment by mikedanese** [comment]
LGTM

**Pull request merged**